### ISSUE TITLE
Arabic Translation - Ultimate consent form - GDPR edition

### ISSUE THREAD

**Issue by Osama-Elyamany** [con/open-brain-consent#136]

**Comment by yarikoptic** [comment]
COOL!  
- would you mind if I add you to https://github.com/orgs/con/teams/arabic ?
- could you please have a look at https://github.com/con/open-brain-consent/pull/59/files which adds translation for the ultimate (non GDPR) -- seems might need a little more work and would be a good example for "translation howto"
- we might need help with https://github.com/con/open-brain-consent/issues/65 if you are strong in dealing with sphinx -- yet to find time myself

**Comment by Osama-Elyamany** [comment]
Thanks!

The first link does not work! :) 

If I got it well, I see in the second link an attempt to add translation in Arabic. Should I proceed though with the Arabic translation of the GDPR edition?

Sorry, actually I am not experienced with Sphinx!

**Comment by yarikoptic** [comment]
re first link: odd... let me rephrase - would you mind if I add you to @con/arabic team?

> If I got it well, I see in the second link an attempt to add translation in Arabic.

correct.

> Should I proceed though with the Arabic translation of the GDPR edition?

I cannot/will not stop anyone from doing good work ;)  But you might benefit from reviewing #59 first to get better sense of language to be used and how reviews would be done etc.
